tls/s2n_cipher_suites.c

```diff
@@ -625,6 +625,20 @@ struct s2n_cipher_suite s2n_dhe_rsa_with_chacha20_poly1305_sha256 = /* 0xCC,0xAA
     .minimum_required_tls_version = S2N_TLS12,
 };
 
+struct s2n_cipher_suite s2n_hybrid_dhe_ecdhe_rsa_with_aes_256_gcm_sha384 = {
+        .available = 0,
+        .name = "HYBRID-DHE-ECDHE-RSA-AES256-GCM-SHA384",
```

Comment:
We are planning to use ECDHE_[KEM] everywhere else for naming. Perhaps this should be ECDHE_DHE to parallel? If we do, the order on the wire should match.